rts/Rml/Backends/RmlUi_Backend.cpp

```diff
@@ -308,6 +308,34 @@ void RmlGui::OnContextDestroy(Rml::Context* context)
 	state->contexts.erase(std::ranges::find(state->contexts, context));
 }
 
+Rml::Context* RmlGui::GetOrCreateContext(const std::string& name)
+{
+	if (!RmlInitialized()) {
+		return nullptr;
+	}
+	
+	Rml::Context* context = Rml::GetContext(name);
+	if (context == nullptr) {
+		context = Rml::CreateContext(name, {0, 0});
+	} else if (state->contexts_to_remove.contains(context)) {
+		state->contexts_to_remove.erase(context);
```

Comment:
Maybe we could do without the `.contains` check? `.erase` has to search for the element anyway so we'd avoid searching it twice if present and won't see any difference if not.

Reply:
I didn't realize I could do that